This fix was published by Paul Howarth on the OpenVPN-devel mailing list
on 2006-09-13 and fixes the TLS test's running indefinitely.

--- ./sample-config-files/loopback-client~	2006-09-12 18:21:43.000000000 +0100
+++ ./sample-config-files/loopback-client	2006-09-12 18:22:40.000000000 +0100
@@ -22,4 +22,4 @@
 cert sample-keys/client.crt
 cipher DES-EDE3-CBC
 ping 1
-inactive 120
+inactive 120 10000000
--- ./sample-config-files/loopback-server~	2006-09-12 18:21:50.000000000 +0100
+++ ./sample-config-files/loopback-server	2006-09-12 18:22:51.000000000 +0100
@@ -23,4 +23,4 @@
 cert sample-keys/server.crt
 cipher DES-EDE3-CBC
 ping 1
-inactive 120
+inactive 120 10000000
